ff/serving/runner/kubernetes.go

```diff
@@ -18,6 +19,54 @@ import (
 
 var namespace string = "default"
 
+type CronSchedule string
+
+func makeCronSchedule(schedule string) (*CronSchedule, error) {
+	if _, err := cronexpr.Parse(schedule); err != nil {
+		return nil, fmt.Errorf("invalid cron expression: %v", err)
+	}
+	cronSchedule := CronSchedule(schedule)
+	return &cronSchedule, nil
+}
+
+func MonthlySchedule(day, minute, hour int) (*CronSchedule, error) {
```

Comment:
Is there an advantage to these over just having a single (day, minute, hour, month, year) function and the ones that can do the */n values?

Reply:
It's a bit more confusing to do it more generally as the user has to have an "any" parameter that necessarily excludes numbers. This way we can enforce integer inputs.